OrbitQt/orbitmainwindow.h

```diff
@@ -95,6 +95,13 @@ class OrbitMainWindow : public QMainWindow {
   // TODO(170468590): [ui beta] When out of ui beta, this can return TargetConfiguration (without
   // std::optional)
   std::optional<orbit_qt::TargetConfiguration> ClearTargetConfiguration() {
```

Comment:
I've just realized this is in the header file - suggest to move the implementation to the cpp file.

Reply:
Yep. Good point. Done.